benchbuild/projects/gentoo/gentoo.py

```diff
@@ -25,13 +25,12 @@
 
 
 @attr.s
-class GentooGroup(project.Project):
+class GentooGroup(bb.Project):
```

Comment:
Codacy found an issue: [Method 'run_tests' is abstract in class 'Project' but is not overridden](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)